aopy/analysis.py

```diff
@@ -292,3 +292,23 @@ def interpolate_extremum_poly2(extremum_idx, data, extrap_peaks=False):
 
     return extremum_time, extremum_value, f
 
+def get_fano_factor_values_per_condition(spiking_data):
```

Comment:
Make function more generic because this isn't actually calculating the fano factor